tka/key.go

```diff
@@ -31,6 +35,24 @@ func (k KeyKind) String() string {
 	}
 }
 
+// MarshalText implements [encoding.TextMarshaler].
+func (k KeyKind) MarshalText() ([]byte, error) {
+	return []byte(k.String()), nil
+}
+
+// UnmarshalText implements [encoding.TextUnmarshaler].
+func (k *KeyKind) UnmarshalText(b []byte) error {
```

Comment:
Do we need unmarshaling support for all of these internal types like AUMKind, KeyKind, Key, etc? Internally we use CBOR to serialize these, and only need JSON for the CLI, right? I would suggest to skip writing code that we won't actually use, and make it explicit that JSON is only used for CLI output.